### PR TITLE
add(epp) : applications-pfsense-fauxapi FR

### PR DESCRIPTION
**Pull request by psamecentreon**

### CODE REVIEW

File: fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+une configuration "très cassée".
+
+## Contenu du Plugin-Pack
+
+### Objets supervisés
+
+* Applications
+* Firewall
+* Router
+* FauxAPI
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:20]: Should be the list of the Services templates no ?

File: fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+### Métriques collectées
+
+Vous pouvez vous renseigner en détails sur les métriques présentées ci-après sur la documentation officielle 
+du Fauxapi Pfsense documentation: https://github.com/ndejong/pfsense_fauxapi
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:25]:
```suggestion
Vous pouvez vous renseigner en détails sur les métriques présentées ci-après sur la documentation officielle 
du Fauxapi Pfsense : https://github.com/ndejong/pfsense_fauxapi
```

File: fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Backup-files-->
+
+| Metric name                                | Description                                  |
+| :----------------------------------------- | :------------------------------------------- |
+| backups.total.count                        | Total number of backups                      |
+| backups.time.last.seconds                  | Last backup time in milliseconds. Unit : ms  |
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:34]:
```suggestion
| backups.time.last.seconds                  | Last backup time in seconds. Unit : s  |
```

File: fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+<!--Gateways-->
+
+| Metric name                                 | Description                                                                       |
+| :------------------------------------------ | :-------------------------------------------------------------------------------- |
+| status                                      | Gateways status                                                                   |
+| gateway.packets.delay.milliseconds          | Delay packets going through the interface in milliseconds. Unit : ms              |
+| gateway.packets.loss.percentage             | Lost packets going through the interface in percentage. Unit : %                  |
+| gateway.packets.stddev.milliseconds         | Standard deviation packets going through the interface in milliseconds. Unit : ms |
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:43]: That's not an interface ;)

File: fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+<!--Rules-->
+
+| Metric name                 | Description                                                         |
+| :-------------------------- | :------------------------------------------------------------------ |
+| rules.total.count           | Total number of rules                                               |
+| rule.traffic.bitspersecond  | Traffic going through the interface in bits per seconds. Unit : b/s |
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:50]: wrong description

File: fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+
+<!--System-->
+
+| Metric name                                | Description                                   |
+| :----------------------------------------- | :-------------------------------------------- |
+| system.connections.tcp.usage.count         | Number of usage TCP connection                |
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:56]:
```suggestion
| system.connections.tcp.usage.count         | Number of TCP connections                |
```

File: fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+| system.connections.tcp.usage.percentage    | Usage TCP connection in percentage. Unit : %  |
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:57]:
```suggestion
| system.connections.tcp.usage.percentage    | Usage TCP connections in percentage. Unit : %  |
```

File: fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+
+Cette commande déclenchera une alarme WARNING si les paquets perdus dépasse les 5% (```--warning-packets-loss='5'```) 
+et une alerte CRITICAL s'il dépasse 10% (```--critical-packets-loss='10'```).
+Des seuils peuvent être positionnés à l'aide des options ```--warning-*``` et ```--critical-*``` sur les métriques.
+
+Toutes les options et leur utilisation peuvent être consultées avec le paramètre ```--help``` ajouté à la commande:
+
+```bash
+/usr/lib/centreon/plugins//centreon_pfsense_fauxapi.pl --plugin=apps::pfsense::fauxapi::plugin 
+--mode=gateways --help
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:159]:
```suggestion
/usr/lib/centreon/plugins//centreon_pfsense_fauxapi.pl --plugin=apps::pfsense::fauxapi::plugin \
--mode=gateways --help
```

File: fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+La commande ci-dessus contrôle le statut de la gateway Pfsense Fauxapi (```--mode=gateways```) nommée *mygateway* (```--filter-name='mygateway'```). 
+Il utilise l'api-key (```--api-key='myapikey'```), l'api-secret (```--api-secret='myapisecret'```)
+et il se connecte à l'hôte _api.pfsensefauxpi.com_ (```--hostname='api.pfsensefauxpi.com'```) 
+sur le port _443_ (```--port='443'```) utilisant le protocol _https_ (```--proto='https'```).
+
+Cette commande déclenchera une alarme CRITICAL si le statut retourné de la gateway est différent de *none* (```--critical-status='%{status} !~ /none/i'```).
+
+Cette commande déclenchera une alarme WARNING si les paquets perdus dépasse les 5% (```--warning-packets-loss='5'```) 
+et une alerte CRITICAL s'il dépasse 10% (```--critical-packets-loss='10'```).
+Des seuils peuvent être positionnés à l'aide des options ```--warning-*``` et ```--critical-*``` sur les métriques.
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md:153]:
```suggestion

Des seuils peuvent être positionnés à l'aide des options ```--warning-*``` et ```--critical-*``` sur les métriques.
```

**Review by pkriko — CHANGES_REQUESTED** [review]

File: fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md (hunk 1 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md]: Done ^^

File: fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md (hunk 4 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md]: Remplaced interface by Pfsense

File: fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md (hunk 5 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md]: Corrected description

**Review by pkriko — APPROVED** [review]

**Pull request merged**